src/reformat.c

```diff
@@ -1241,8 +1241,11 @@ static avifResult avifRGBImageToF16(avifRGBImage * rgb)
 static avifResult avifImageYUVToRGBImpl(const avifImage * image, avifRGBImage * rgb, avifReformatState * state, avifAlphaMultiplyMode alphaMultiplyMode)
 {
     avifBool convertedWithLibYUV = AVIF_FALSE;
+    avifBool reformatAlpha = avifRGBFormatHasAlpha(rgb->format) &&
```

Comment:
Move the comment at line 1258 (`// Reformat alpha, if user asks for it, or (un)multiply processing needs it.`) before this line.